src/pages/CreateEvent/EventForm.tsx

```diff
@@ -163,230 +232,373 @@ const EventForm: React.FC = () => {
-                  className={`form-control ${
-                    errors.announcementDate ? "error" : ""
-                  }`}
-                />
-              )}
-            />
-            <span className="divider">|</span>
-            <ClockIcon className="icon" />
-            <Controller
-              name="announcementTime"
-              control={control}
-              rules={{ required: "Announcement time is required" }}
-              render={({ field }) => (
-                <input
-                  type="time"
-                  {...field}
-                  className={`form-control ${
-                    errors.announcementTime ? "error" : ""
-                  }`}
-                />
-              )}
-            />
-          </div>
-          {errors.announcementDate && (
-            <p className="error-text">{errors.announcementDate.message}</p>
-          )}
+          {errors.endDate && <p className="error">{errors.endDate.message}</p>}
+          {errors.endTime && <p className="error">{errors.endTime.message}</p>}
         </div>
 
         <div className="form-group">
-          <label>Timezone</label>
+          <label>Timezone<sup>*</sup></label>
           <select
             {...register("timezone", { required: "Timezone is required" })}
+            className={`form-control${errors.timezone ? " error" : ""}`}
           >
+            <option value="">Select a timezone</option>
             <option value="GMT-0700">PST (GMT-0700)</option>
+            <option value="GMT-0600">MST (GMT-0600)</option>
+            <option value="GMT-0600">CST (GMT-0600)</option>
             <option value="GMT-0500">EST (GMT-0500)</option>
           </select>
+          {errors.timezone && (
+            <p className="error">{errors.timezone.message}</p>
+          )}
         </div>
 
         <div className="form-group">
-          <label>Meeting Link *</label>
-          <input type="url" {...register("meetingLink")} />
+          <label>Meeting Link<sup>*</sup></label>
+          <input
+            type="url"
+            {...register("meetingLink", {
+              required: "Meeting Link is required",
+            })}
+            className={`form-control${errors.meetingLink ? " error" : ""}`}
+          />
+          {errors.meetingLink && (
+            <p className="error">{errors.meetingLink.message}</p>
+          )}
         </div>
 
         <div className="form-group">
-          <label>Participant Count *</label>
-          <input
-            type="number"
-            {...register("minParticipants", { valueAsNumber: true })}
-            placeholder="Min"
-          />{" "}
-          -
-          <input
-            type="number"
-            {...register("maxParticipants", { valueAsNumber: true })}
-            placeholder="Max"
-          />
+          <label>Participant Count<sup>*</sup></label>
+          <div className="flex flex-col">
+            <label>Min</label>
+            <input
```

Comment:
organizers should not be allowed to 0 participant count or less than 0

Reply:
Done